x-pack/plugins/lens/public/indexpattern_datasource/fields_accordion.tsx

```diff
@@ -69,73 +70,81 @@ export const InnerFieldsAccordion = function InnerFieldsAccordion({
   exists,
   hideDetails,
   showExistenceFetchError,
+  groupIndex,
   dropOntoWorkspace,
   hasSuggestionForField,
 }: FieldsAccordionProps) {
   const renderField = useCallback(
-    (field: IndexPatternField) => (
+    (field: IndexPatternField, index) => (
       <FieldItem
         {...fieldProps}
         key={field.name}
         field={field}
         exists={exists(field)}
         hideDetails={hideDetails}
+        itemIndex={index}
```

Comment:
I guess those will be needed later for ordering the drop targets?

Reply:
Yes! I will just left them here hanging if you don't mind!